Polyfills/Canvas/Source/Canvas.cpp

```diff
@@ -114,8 +114,17 @@ namespace Babylon::Polyfills::Internal
     {
         if (m_dirty)
         {
+            // make sure render targets are filled with 0 : https://registry.khronos.org/webgl/specs/latest/1.0/#TEXIMAGE2D
```

Comment:
nit: Maybe this can be shared with the Graphics version at some point? No need to block the PR, but just noting it.

Reply:
Yes, maybe move framebuffer creation in Graphics  as well.